modules/wkt/src/lib/encode-wkt.ts

```diff
@@ -4,28 +4,40 @@
 /**
  * Stringifies a GeoJSON object into WKT
  */
-export default function encodeWKT(gj) {
+
+/**
+ * @param gj
+ * @returns string
+ */
+
+export default function encodeWKT(gj: {
+  type: string;
+  properties?: {};
+  geometry?: any;
+  coordinates?: any;
+  geometries?: any;
+}) {
   if (gj.type === 'Feature') {
     gj = gj.geometry;
   }
 
-  function pairWKT(c) {
+  function pairWKT(c: any[]) {
```

Comment:
Should we detect what particular type here?
Do they always have different types?